### PR TITLE
Fix raw_reads count extraction.

### PR DESCRIPTION
**Pull request by charles-cowart**
Simplest fix. Assumes we can always multiply the number of reads by 2. Does not pull raw_reads from fastp JSON files.
raw_reads count extraction previously was accounting only for half of the reads.

### CODE REVIEW

**Comment by charles-cowart** [comment]
@antgonza @mmbryant23 ready for review!

File: metapool/count.py [hunk 1]
```diff
@@ -231,10 +236,10 @@ def run_counts(run_dir, metadata):
             minimap2_counts(run_dir, metadata)])
 
     # convenience columns to assess sample quality
-    out['fraction_passing_quality_filter'] = (out['quality_filtered_reads'] /
-                                              out['raw_reads'])
+    good_ratio = out['quality_filtered_reads_r1r2'] / out['raw_reads_r1r2']
```
Comment (antgonza) [inline metapool/count.py:239]: why is this good and where is the bad ratio?

File: metapool/count.py [hunk 2]
```diff
@@ -231,10 +236,10 @@ def run_counts(run_dir, metadata):
...
+    out['fraction_passing_quality_filter'] = good_ratio
```
Comment (antgonza) [inline metapool/count.py:240]: why split in 2 lines?

**Review by antgonza — COMMENTED** [review]
Minor comments.

**Review by mmbryant23 — APPROVED** [review]

File: metapool/count.py (hunk 2 above)
Comment (charles-cowart) [reply metapool/count.py:240]: Point taken. :) renamed to 'ratio'.

File: metapool/count.py (hunk 1 above)
Comment (charles-cowart) [reply metapool/count.py:239]: Point taken. :) renamed to 'ratio'.

**Comment by antgonza** [comment]
Fantastic, thank you! I'll wait to the qiita PR to be merged (after files are renamed) and this PR is cleaned up accordingly!

**Pull request merged**